### PR TITLE
Update test.md

### PR DESCRIPTION
**Pull request by ayebbp**

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -1,3 +1 @@
 new update codes
-fafafaf
-စော်ဖိုက်ချင်တယ် အိုကေ
```
Comment (ayarhlaine) [inline test.md:3]: Why you delete this two lines?

**Review by ayarhlaine — COMMENTED** [review]
REP:

File: test.md (hunk 1 above)
Comment (ayebbp) [reply test.md:3]: REASON:

**Review by ayarhlaine — APPROVED** [review]
Approve

**Pull request merged**